developer/wip/for-contributors/control-plane/protocol-extensions.md

```diff
@@ -0,0 +1,94 @@
+# Protocol Extensions
+
+<!-- TOC -->
+* [Protocol Extensions](#protocol-extensions)
+  * [1. RemoteMessage](#1-remotemessage)
+    * [1.1 Delivering messages with RemoteMessageDispatcher](#11-delivering-messages-with-remotemessagedispatcher)   
+    * [1.2 Handling incoming messages with protocol services](#12-handling-incoming-messages-with-protocol-services)   
+  * [2. DSP protocol](#2-dsp-protocol-implementation)
+    * [2.1 `RemoteMessage` handlers](#21-remotemessage-handlers)
+    * [2.2 HTTP endpoints](#22-http-endpoints)
+    * [2.3 `RemoteMessage` transformers](#22-remotemessage-transformers)
+
+<!-- TOC -->
+
+
+The EDC officially supports the [Dataspace protocol](#2-dsp-protocol) using the HTTPs bindings, but since it is an extensible platform, multiple protocol implementations can be supported for inter-connectors communication. Each supported protocols is identified by a unique key used by EDC for dispatching a [remote message](#1-remotemessage). 
```

Comment:
```suggestion
The EDC officially supports the [Dataspace protocol](#2-dsp-protocol) using the HTTPs bindings, but since it is an extensible platform, multiple protocol implementations can be supported for inter-connectors communication. Each supported protocol is identified by a unique key used by EDC for dispatching a [remote message](#1-remotemessage). 
```